### PR TITLE
add features to output config

### PR DESCRIPTION
**Pull request by soyvural**
- add features to output config

### CODE REVIEW

**Comment by soyvural** [comment]
@zquiring001 could you please review this PR?

File: configuration/outputs.md [hunk 1]
```diff
@@ -146,6 +152,7 @@ If enabled, the Scalyr integration will stream analytics and insights to your Sc
 | name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
 | type | Streaming destination type \(i.e. scalyr, influxdb, sumologic, datadog, etc.\) | Yes |
 | endpoint | Scalyr endpoint | Yes |
+| features | Features defines which data types stream to backend. Only "log" is supported. | No |
```
Comment (zquiring001) [inline configuration/outputs.md]: I thought Scalyr supported both logs + metrics, but the metrics come in via the logs (uploaddata) api?

Comment (soyvural) [reply configuration/outputs.md]: it is corrected, thanks

**Comment by yildizfatih** [comment]
@soyvural - is merge pending any decision?

**Pull request merged**